portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudIndividualTests.testcase

```diff
@@ -0,0 +1,75 @@
+@component-name = "portal-analytics-cloud"
+definition {
+
+	property analytics.cloud.enabled = "true";
+	property minimum.slave.ram = "32";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Analytics Cloud";
+
+	setUp {
+		ACUtils.launchAC();
+
+		ACSettings.takeTokenFromAnalyticsCloud();
+
+		TestCase.setUpPortalInstance();
+
+		Navigator.openURL();
+
+		User.firstLoginUI();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		Navigator.openURL();
+
+		ACDXPSettings.disconnectDXPFromAnalyticsCloud();
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Set custom range filter for known individuals"
+	@priority = "5"
+	test CanSetCustomRangeKnownIndividuals {
```

Comment:
hi @calvinctvo , I think the test name could be `CanSetEmptyDataCustomRangeOnIndividualActivitiesCard`